pkg/daemon/ceph/client/osd.go

```diff
@@ -259,7 +260,10 @@ func OSDRemove(context *clusterd.Context, clusterName string, osdID int) (string
 	return string(buf), err
 }
 
-func OsdSafeToDestroy(context *clusterd.Context, clusterName string, osdID int) (bool, error) {
+func OsdSafeToDestroy(context *clusterd.Context, clusterName string, osdID int, cephVersion cephver.CephVersion) (bool, error) {
+	if !cephVersion.IsAtLeastNautilus() {
+		return false, fmt.Errorf("failed to get safe-to-destroy status: ceph version in lower than Nautilus")
```

Comment:
Returning an error is going to result in logs of log messages. I'd suggest returning `false, nil` and just add a debug log message that the feature is disabled.

Reply:
updated